spec.md

```diff
@@ -3456,11 +3470,21 @@ digit_hex = digit_dec
           | "A" | "B" | "C" | "D" | "E" | "F"
           | "a" | "b" | "c" | "d" | "e" | "f" ;
 
-(* An integer *)
-int = '"' , "b" , [ "-" ] , { digit_bin } , '"'
-    | '"' , "o" , [ "-" ] , { digit_oct } , '"'
-    | '"' , "h" , [ "-" ] , { digit_hex } , '"'
-    |             [ "-" ] , { digit_bin } ;
+(* Integer Literals *)
+int = [ "-" ] , { digit_bin } ;
+
+(* String-encoded Integer Literals *)
+int_se =
+    '"' , "b" , [ "-" ] , { digit_bin } , '"'
+  | '"' , "o" , [ "-" ] , { digit_oct } , '"'
+  | '"' , "h" , [ "-" ] , { digit_hex } , '"' ;
+
+(* Any Integer Literal *)
+int_any = int | int_se ;
+
+(* String Literals *)
+string = (* a string that is not defined and assumed to exist *) ;
```

Comment:
This is weird to me.. can you clarify?

Reply:
I don't know how to define "string" nor do I want to. Other places that I've seen this, e.g., in the Python grammar, they will use something external, like `STRING`. What I'm trying to communicate here is that a person reading this needs to assume the existence of a string and/or this is not specified within the scope of this document.

Reply:
stupid question, why not actually define this? Do we allow unicode characters? Snowmen and smiley faces? can we point to some external definition of a string? How to escape 'quotes' in the middle of strings? It seems like there would probably be some standard string definitions to use?